src/mnelab/dialogs/__init__.py

```diff
@@ -7,6 +7,7 @@
 from mnelab.dialogs.brainvision import BrainVisionDialog
 from mnelab.dialogs.calc import CalcDialog
 from mnelab.dialogs.channel_properties import ChannelPropertiesDialog
+from mnelab.dialogs.create_annotations_from_events import AnnotationsIntervalDialog
```

Comment:
I think calling the module `annotations_from_events` is sufficient.

```suggestion
from mnelab.dialogs.annotations_from_events import AnnotationsIntervalDialog
```